docs/developer-guide/maps-configuration.md

```diff
@@ -728,6 +728,144 @@ The `style` or `styleName` properties of vector layers (wfs, vector...) allow to
+
+```json
+{
+        "type": "vector",
+        "visibility": true,
+        "id": "1234",
+        "name": "1234",
+        "hideLoading": true,
+        "features": [
+          {
+            "type": "Feature",
+            "geometry": {
+              "type": "Point",
+              "coordinates": [2,0]
+            },
+            "properties": {},
+            "style": [
+              {
+                "iconAnchor": [0.5,0.5],
+                "anchorXUnits": "fraction",
+                "anchorYUnits": "fraction",
+                "opacity": 1,
+                "size": 30,
+                "symbolUrl": "data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='30' height='30'%3E%3Crect  x='5' y='5' width='20' height='20' style='fill:rgb(255,0,0);stroke-width:5;stroke:rgb(0,0,0)' /%3E%3C/svg%3E",
+                "shape": "triangle",
+                "id": "c65cadc0-9b46-11ea-a138-dd5f1faf9a0d",
+                "highlight": false,
+                "weight": 4
+              }
+            ]
+          },{
+            "type": "Feature",
+            "geometry": {
+              "type": "Polygon",
+              "coordinates": [[[0, 0],[1, 0],[1, 1],[0,1],[ 0,0]]]
+            },
+            "properties": {},
+            "style": [
+              {
+                "color": "#d0021b",
+                "opacity": 1,
+                "weight": 3,
+                "fillColor": "#4a90e2",
+                "fillOpacity": 0.5,
+                "highlight": false,
+                "dashArray": ["6","6"]
+              }
+            ]
+          },{
+            "type": "Feature",
+            "geometry": {
+              "coordinates": [[0, 2],[ 0,3]],
+              "type": "LineString"
+            },
+            "properties": {},
+            "style": [
+              {
+                "color": "#ffcc33",
+                "opacity": 1,
+                "weight": 3,
+                "editing": {
+                  "fill": 1
+                },
+                "highlight": false
+              },
+              {
+                "iconGlyph": "comment",
+                "iconShape": "square",
+                "iconColor": "blue",
+                "highlight": false,
+                "iconAnchor": [ 0.5,0.5],
+                "type": "Point",
+                "title": "StartPoint Style",
+                "geometry": "startPoint",
+                "filtering": true
+              },
+              {
+                "iconGlyph": "shopping-cart",
+                "iconShape": "circle",
+                "iconColor": "red",
+                "highlight": false,
+                "iconAnchor": [ 0.5,0.5 ],
+                "type": "Point",
+                "title": "EndPoint Style",
+                "geometry": "endPoint",
+                "filtering": true
+              }
+            ]
+          }
+        ]
+      }
+```
+
+*Result:*
+
+<img src="../img/vector-style-annotations.jpg" class="ms-docimage"  style="max-width:600px;"/>
```

Comment:
you sure that this path works when deployed?

Reply:
it is used here and there in doc. I tested it with mkdocs serve (it is relative so it should be ok). There is some problem you encountered about this ?

Reply:
sounds good